UI UX finished

salirezanm opened the pull request:
please pull this branch to you'r repository

salirezanm committed:
change something in ui

salirezanm committed:
added Picasso dependency

salirezanm committed:
get poster form server and show it

salirezanm committed:
change somethings in ui

salirezanm committed:
Merge branch 'code' of https://github.com/imansdn/moviesApp into code
change somthings in ui

salirezanm committed:
nothing!

salirezanm committed:
add picasso dependency

salirezanm committed:
add picasso dependency

salirezanm committed:
add picasso dependency

salirezanm committed:
get poster from server and show it

salirezanm committed:
Merge remote-tracking branch 'origin/code' into code

# Conflicts:
#	app/build.gradle

salirezanm committed:
added picasso dependency

salirezanm committed:
idn2

salirezanm committed:
added picasso dependency and get poster from server to show it

salirezanm committed:
get year of movie from server to show it

salirezanm committed:
get country of movie from server to show it

salirezanm committed:
solved a conflict

salirezanm committed:
get IMDB rating of movie from server to show it

salirezanm committed:
get genres of movie from server to show it

imansdn commented:
please wait

imansdn committed:
add recyclerView dependency

imansdn committed:
create item xml for movie

imansdn committed:
add recyclerView on main activity xml

imansdn committed:
create movie Adapter for recycler

imansdn committed:
get movies from server and pass them to adapter
and notify adapter to update itself

imansdn commented:
now you can pull this branch and continue the development.
no need to merge to master yet.

salirezanm committed:
added recyclerview-animator and matarial-ripple dependency

salirezanm committed:
added animation for recyclerview items

salirezanm committed:
added ripple animation for recyclerview items

salirezanm committed:
added searchMovies API

salirezanm committed:
added search editText

salirezanm committed:
Replace RelativeLayout with ConstraintLayout in items.xml and MainActivity.xml .
some change in xml file for syncing with ConstraintLayout .
added: {
singleLine -> true
ellipsize -> marquee
marqueeRepeatLimit -> marquee_forever
focusable -> true
focusableInTouchMode -> true
scrollHorizontally -> true
} for txt_title & txt_country & txt_genres .

imansdn committed:
add search icon to drawable - note! please install material design icons plugin for see icons

imansdn committed:
add image view next to search edit Text

imansdn committed:
define editText

imansdn committed:
add a Todo for EndLess

salirezanm committed:
define Tablayout and viewPager in MainActivity.java & xml .

define MainPageFragment and added code for getting data from server and set on views .

define ViewPagerAdapter.java

define edittext for searching movie.

salirezanm committed:
Merge remote-tracking branch 'origin/code' into code

# Conflicts:
#	app/src/main/java/ir/imandroid/moviesapp/activity/MainActivity.java
#	app/src/main/res/layout/activity_main.xml

imansdn committed:
add search logo

imansdn committed:
set search logo in imageView

imansdn committed:
add a Todo for EndLess

imansdn committed:
add a Todo for EndLess

salirezanm committed:
create SearchFragment for search movies

salirezanm committed:
fix a mistake in MainActivity .
testing somethings in MainPageFragment (nit complete yet) .
set view global and testing somethings with several toast in SearchFragment.

salirezanm committed:
convert FrameLayout to ConstraintLayout in SearchFragment

salirezanm committed:
Displacement imageView & edt_search from MainPageFragment.xml to SearchFragment.xml

salirezanm committed:
added search API to service

salirezanm committed:
setting search fragment to search on movies of server

salirezanm committed:
added recylerview for in fragment_search.xml

salirezanm committed:
added GenresFragment .

salirezanm committed:
added string name of tab to string resources values

salirezanm committed:
setup this frament for get all genres go movies from server

salirezanm committed:
change FrameLayout to ConstraintLayout

salirezanm committed:
added value 20sp size for txt_genres in fragment_genres.xml for show genres of movies .

imansdn committed:
no

salirezanm committed:
add moreInfoActivity

salirezanm committed:
added animation collapsing toolbar android(not complete and finaly)

salirezanm committed:
setting animation collapsing toolbar android (not complete)

imansdn committed:
create model for add movie response

imansdn committed:
create add movie request

imansdn committed:
create method to call a retrofit request to add a movie

salirezanm committed:
added solotion for adding movies

salirezanm committed:
fixing CollapsingToolbarLayout (just for show, not complete for showing data)

imansdn committed:
add fab button and coordinator layout to main fragment

imansdn committed:
remove some extra fields from constant

imansdn committed:
replace butterKnife with findViewById in main fragment.
add onclick to fab button and put a request for add a movie inside that.
move some define variables to onCreate method instead of onCreateView

imansdn committed:
put add icon to drawable

imansdn committed:
write some TODO items

salirezanm committed:
get data from server code

salirezanm committed:
casting imageView img_movie_poster

salirezanm committed:
get postion from adapter

salirezanm committed:
get movie poster from server

salirezanm committed:
putExtra position value to MoreInfoActivity

salirezanm committed:
set img_movie_poster id

salirezanm committed:
get movie data from server and add views for display it,s

salirezanm committed:
fix some issue on nested scroll view

salirezanm committed:
added views for display extra movie info in content_collapsible_toolbar.xml

salirezanm committed:
some string data in strings.xml

salirezanm committed:
nothing!

salirezanm committed:
Get photos of movie scenes and set on imageViews

salirezanm committed:
remove some testing code

salirezanm committed:
Added rounded image view library

salirezanm committed:
Added scrollView for Photos of movie scenes.
replace imageViews with RoundedImageViews.

salirezanm committed:
testing git kraken

imansdn committed:
delete moreInfoActivity

imansdn committed:
create moreInfoFragment

imansdn committed:
create BaseFragment

imansdn committed:
extends and implement all fragments from BaseFragments

imansdn committed:
handle on back pressd on MainActivity and detect selected fragment

imansdn committed:
replace current fragment with moreInfoFragment after click on any position to show more info
